Makefile

```diff
@@ -1,12 +1,11 @@
-CURL=curl -sSf
+CURL:=curl -sSf
 
-IMAGE_NAME := quay.io/coreos/flannel-cni
-VERSION ?= latest
-TAG ?= 0.1
-CNI_VERSION ?= v0.5.2
+FLANNEL_CNI_ROOT:=$(shell git rev-parse --show-toplevel)
+IMAGE_NAME:=quay.io/coreos/flannel-cni
+VERSION:=$(shell $FLANNEL_CNI_ROOT/scripts/git-version)
```

Comment:
why are there two different versions? The aim of the git-version script is to automatically read the version as a Git SHA hash or Git tag (if present). Seems like you'd either want to use that tagging pattern or set the version directly, not both.